test/verify/storagelib.py

```diff
@@ -363,6 +363,19 @@ def teardown():
             self.dialog_cancel()
             self.dialog_wait_close()
         self.retry(setup, check, teardown)
+
+    def dialog_apply_with_retry(self):
+        def step():
+            try:
+                self.dialog_apply()
+                self.dialog_wait_close()
+            except Error as ex:
+                return False
+            return True
+        self.browser.wait(step)
```

Comment:
You just need wait here I think, not browser.wait.
Both are having some timeout and wait for the function to return truthy value, it's not harmful, just confusing.

Reply:
PS: Also with the other wait you can set the tries and delay variables as you wish.

Reply:
The rest of the code uses `browser.wait`, so I'd say we use it here, too.

(There used to be a difference.  `browser.wait` would spin the Browser event loop, while just `wait` would not.  With the move away from phantomjs, this difference has disappeared, so we could indeed clean this up globally.)

Reply:
I think the intended difference was that `Browser.wait()` would get a JS condition to be evaluated in the browser page, while `wait()` would wait for something on the VM, like a shell command to become true.